### PR TITLE
Load ERCC counts from S3 and fix RPM

### PR DESCRIPTION
**Pull request by cdebourcy**

### CODE REVIEW

File: app/lib/report_helper.rb [hunk 1]
```diff
@@ -195,7 +195,8 @@ def all_categories
 
   def fetch_taxon_counts(pipeline_run_id, background_id)
     pipeline_run = PipelineRun.find(pipeline_run_id)
-    adjusted_total_reads = pipeline_run.total_reads * pipeline_run.subsample_fraction
+    total_ercc_reads = pipeline_run.ercc_counts.map(&:count).sum # will be zero if no ercc_counts
```
Comment (yunfangjuan) [inline app/lib/report_helper.rb]: we should just put the total in a new pipeline_run field so we can avoid querying database every time for this.

**Comment by yunfangjuan** [comment]
you probably want to write a quick rake script to load the RR007 reads_per_gene.star.tab into the database.

File: app/models/pipeline_run_stage.rb [hunk 2]
```diff
@@ -234,20 +234,33 @@ def postprocess_command
...
+    # Load version
     version_s3_path = "#{pr.sample_output_s3_path}/#{PipelineRun::VERSION_JSON_NAME}"
     pr.version = `aws s3 cp #{version_s3_path} -`
 
-    # TODO(yf): remove the following line
-    pr.job_stats_attributes = stats_array
-
-    # rm the json
-    _stdout, _stderr, _status = Open3.capture3("rm -f #{downloaded_stats_path}")
+    # Load ERCC counts
```
Comment (yunfangjuan) [inline app/models/pipeline_run_stage.rb]: 251-263 should go into its own function.

File: app/lib/report_helper.rb (hunk 1 above)
Comment (cdebourcy) [reply app/lib/report_helper.rb]: ok

**Pull request merged**